samples/net/echo_client/sample.yaml

```diff
@@ -16,8 +16,8 @@ tests:
     extra_args: CONF_FILE="prj_bt.conf"
     platform_whitelist: qemu_x86
     tags: bluetooth
-  test_cc2520:
-    extra_args: CONF_FILE="prj_frdm_k64f_cc2520.conf"
+  test_mcr20a:
+    extra_args: CONF_FILE="prj_frdm_k64f_mcr20a.conf"
```

Comment:
This seems like we should have any entry for prj_frdm_k64f_cc2520.conf and one for prj_frdm_k64f_mcr20a.conf

Reply:
If we really are removing support for prj_frdm_k64f_cc2520.conf, we should remove the file as well.

Reply:
We could indeed. Btw: how to apply dts in samples?